modules/installation-guide/examples/system-variables.adoc

```diff
@@ -168,39 +166,39 @@ pass:[<!-- vale off -->]
+ `+CHE_WORKSPACE_DEVFILE__REGISTRY__URL+`,"`+https://che-devfile-registry.prod-preview.openshift.io/+`","Devfile Registry endpoint. Should be a valid HTTP URL. Example: ++http://che-devfile-registry-eclipse-che.192.168.65.2.nip.io++ In case {prod-short} plug-ins registry is not needed value 'NULL' should be used" 
+ `+CHE_WORKSPACE_DEVFILE__REGISTRY__INTERNAL__URL+`,"`+NULL+`","Devfile Registry 'internal' endpoint. Should be a valid HTTP URL. Example: ++http://plugin-registry.che.svc.cluster.local:8080++ In case {prod-short} plug-ins registry is not needed value 'NULL' should be used" 
+ `+CHE_WORKSPACE_STORAGE_AVAILABLE__TYPES+`,"`+persistent,ephemeral,async+`","The configuration property that defines available values for storage types that clients such as the Dashboard should propose to users during workspace creation and update. Available values:   - `persistent`: Persistent Storage slow I/O but persistent.   - `ephemeral`: Ephemeral Storage allows for faster I/O but may have limited storage       and is not persistent.   - `async`: Experimental feature: Asynchronous storage is combination of Ephemeral       and Persistent storage. Allows for faster I/O and keep your changes, will backup on stop       and restore on start workspace.       Will work only if:           - `che.infra.kubernetes.pvc.strategy='common'`           - `che.limits.user.workspaces.run.count=1`           - `che.infra.kubernetes.namespace.allow_user_defined=false`           - `che.infra.kubernetes.namespace.default` contains `<username>`      in other cases remove `async` from the list." 
+ `+CHE_WORKSPACE_STORAGE_PREFERRED__TYPE+`,"`+persistent+`","The configuration property that defines a default value for storage type that clients such as the Dashboard should propose to users during workspace creation and update. The `async` value is an experimental feature, not recommended as default type." 
+ `+CHE_SERVER_SECURE__EXPOSER+`,"`+jwtproxy+`","Configures in which way secure servers will be protected with authentication. Suitable values:   - `default`: `jwtproxy` is configured in a pass-through mode. Servers should authenticate requests themselves.   - `jwtproxy`: `jwtproxy` will authenticate requests. Servers will receive only authenticated requests." 
+ `+CHE_SERVER_SECURE__EXPOSER_JWTPROXY_TOKEN_ISSUER+`,"`+wsmaster+`","`Jwtproxy` issuer string, token lifetime, and optional auth page path to route unsigned requests to." 
+ `+CHE_SERVER_SECURE__EXPOSER_JWTPROXY_TOKEN_TTL+`,"`+8800h+`","JWTProxy issuer token lifetime." 
+ `+CHE_SERVER_SECURE__EXPOSER_JWTPROXY_AUTH_LOADER_PATH+`,"`+/_app/loader.html+`","Optional authentication page path to route unsigned requests to." 
+ `+CHE_SERVER_SECURE__EXPOSER_JWTPROXY_IMAGE+`,"`+quay.io/eclipse/che-jwtproxy:0.10.0+`","JWTProxy image." 
```

Comment:
f this is upstream doc only, great. If this is downstreamed, then the CHE_SERVER_SECURE__EXPOSER_JWTPROXY_IMAGE has a different value in CRW.

Latest CRW jwtproxy image is https://catalog.redhat.com/software/containers/codeready-workspaces/jwtproxy-rhel8/5da62cd1d70cc5164489e7a9